recipes/AdaptiveCpp/meta.yaml

```
@@ -0,0 +1,73 @@
+{% set name = "AdaptiveCpp" %}
+{% set version = "24.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/AdaptiveCpp/AdaptiveCpp/archive/v{{ version }}.tar.gz
+    sha256: 3bcd94eee41adea3ccc58390498ec9fd30e1548af5330a319be8ce3e034a6a0b
+    patches:
+      - patches/fallback-missing-functions.patch  # [linux]
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('adaptivecpp', max_pin='x.x') }}
+  ignore_run_exports:
+    - libboost
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - ninja  # [not win]
+  host:
+    - clangdev
+    - libboost-devel
+    - llvm-openmp >=14,<19
+    - llvm >=14,<19
+    - spirv-tools
+  run:
+    - llvm >=14,<19
+
+test:
+  commands:
+    - test -f $PREFIX/include/AdaptiveCpp/sycl/sycl.hpp  # [not win]
+    - test -f $PREFIX/lib/cmake/AdaptiveCpp/adaptivecpp-config.cmake  # [not win]
+    - test -f $PREFIX/lib/libacpp-common${SHLIB_EXT}  # [not win]
+    - $PREFIX/bin/acpp-info  # [not win]
+    - if not exist %PREFIX%\\Library\\include\\AdaptiveCpp\\SYCL\\sycl.hpp exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\lib\\cmake\\AdaptiveCpp\\adaptivecpp-config.cmake exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\lib\\acpp-common.lib exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\bin\\acpp-common.dll exit 1  # [win]
+    - "%PREFIX%\\Library\\bin\\acpp-info.exe"  # [win]
```

```suggestion
    - acpp-info  # [not win]
    - if not exist %PREFIX%\\Library\\include\\AdaptiveCpp\\SYCL\\sycl.hpp exit 1  # [win]
    - if not exist %PREFIX%\\Library\\lib\\cmake\\AdaptiveCpp\\adaptivecpp-config.cmake exit 1  # [win]
    - if not exist %PREFIX%\\Library\\lib\\acpp-common.lib exit 1  # [win]
    - if not exist %PREFIX%\\Library\\bin\\acpp-common.dll exit 1  # [win]
    - acpp-info.exe  # [win]
```
If these CLIs are installed correctly, they should be on the path and they shouldn't need to be accessed using the full name.

---

Good to know, thanks!